### PR TITLE
[DOCS] Adjust Invalidate Token REST API docs

### PR DESCRIPTION
**Pull request by jkakavas**
- Renames API to Invalidate Token ( from Delete Token )
- Explicitly calls out the possibility to invalidate refresh tokens
via this API

### CODE REVIEW

**Review by lcawl — APPROVED** [review]
LGTM

File: x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
-pertain to deleting a token:
+pertain to invalidating a token:
+
+`token` (optional)::
+(string) An access token. This parameter cannot be used when `refresh_token` is used.
 
-`token` (required)::
-(string) An access token.
+`refresh_token` (optional)::
+(string) A refresh token. This parameter cannot be used when `token` is used.
```
Comment (tvernum) [inline x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc]: It's obvious to us, but sometimes things are not so obvious to others.... should we mention that one of them is required?
_Although I see, technically, if you don't specify either, then it's just a no-op action_.

**Review by tvernum — APPROVED** [review]
LGTM, with 1 optional suggestion.

File: x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc (hunk 1 above)
Comment (jkakavas) [reply x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc]: Yes, I also contemplated how this should be written and didn't find any examples of mutually excluded parameters for any of our APIs.. I'd better add a NOTE and clarify this

**Pull request merged**